tcms/core/tests/test_markdown.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+import unittest
+import bleach
+
+from tcms.core.templatetags.extra_filters import markdown2html
+
+
+class TestMarkdownExtraFilters(unittest.TestCase):
+
+    def setUp(self):
+        self.md_simple_text = markdown2html("__*hello!*__")
+        self.md_table = markdown2html("""|Stx |Desc |\n|----|-----|\n|Head|Title|\n|Text|Text |""")
+        self.md_nl2br = markdown2html("""Line 1

```

Comment:
what does `nl2br` stand for? I am really confused by this variable names. unless it's something really common in markdown that I don't know of, maybe it's better to change it to something more clear